tests/test.js

```diff
@@ -156,6 +156,15 @@ test("non-string input", () => {
 	assert.equal(pc.red(Infinity), FMT.red[0] + "Infinity" + FMT.red[1])
 })
 
+test("shouldn't overflow when coloring already colored large text", () => {
+  try {
+    pc.blue(pc.red("x").repeat(10000))
+    assert(true)
+  } catch {
```

Comment:
Omitting `(error)` only works in newer Node.js versions. Could you please add `(error)` so the rest of pipelines would go green? Thanks

Reply:
Oops! 
Didn't check the CI tests.
Thank you! 

Now everything is green :)